cmd/pr-helper/entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -x
```

Comment:
Sure you don't want
`  -e  Exit immediately if a command exits with a non-zero status.`
Up to you ofc

Reply:
I wanted to avoid failing if ln fails. It should never since even if the linked file doesn't exist it simply creates the symlink. The only case, if the symlink already exists, which should never be the case

Reply:
You can set -e before that line, the important bit is that the qemu-pre-helper invocation doesn't fail, right?

Reply:
Righ! Let's do it like this!